homeassistant/components/blueprint/importer.py

```diff
@@ -0,0 +1,177 @@
+"""Import logic for blueprint."""
+from dataclasses import dataclass
+import re
+from typing import Optional
+
+import voluptuous as vol
+import yarl
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import aiohttp_client, config_validation as cv
+from homeassistant.util import yaml
+
+from .models import Blueprint
+from .schemas import is_blueprint_config
+
+COMMUNITY_TOPIC_PATTERN = re.compile(
+    r"^https://community.home-assistant.io/t/[a-z-]+/(?P<topic>\d+)(?:/(?P<post>\d+)|)$"
```

Comment:
The topic slug may contain numbers too

```suggestion
    r"^https://community.home-assistant.io/t/[a-z0-9-]+/(?P<topic>\d+)(?:/(?P<post>\d+)|)$"
```